roles/web/templates/nginx.conf.j2

```diff
@@ -48,7 +48,7 @@ http {
 
     server {
         listen       1443;
-        server_name  {{ domain_name }};
+        server_name  {{ domain_name }} meta-{{ domain_name }};
```

Comment:
It's useful to be able to connect directly to the instance for debugging purposes, and have nginx treat it as a known hostname.